include/zephyr/bluetooth/mesh/brg_cfg.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_H__
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup bt_mesh_brg_cfg Bridge Configuration common header
```

Comment:
```suggestion
 * @defgroup bt_mesh_brg_cfg Subnetwork Bridge Configuration common header
```

Reply:
Will keep the name from the Mesh Protocol spec.